app/src/test/java/de/westnordost/streetcomplete/data/visiblequests/QuestPresetControllerTest.kt

```diff
@@ -55,12 +56,12 @@ class QuestPresetControllerTest {
         ctrl.delete(55)
         verify(questPresetsDao).delete(55)
         verify(listener).onDeletedQuestPreset(55)
-        verify(selectedQuestPresetStore).set(0)
+        verify(prefs).selectedQuestPreset = 0L
     }
 
     @Test fun `change current preset`() {
```

Comment:
It seems like this test now fails on the master branch (even with the [post-merge fix commit `6ba0371`](https://github.com/streetcomplete/StreetComplete/commit/6ba0371a3a9da23ee79b91b4f6606739127fc72d)): https://github.com/streetcomplete/StreetComplete/actions/runs/9829796567/job/27135248550

Reply:
Thanks, I will look into it